Criando três novas API's

vicnetto opened the pull request:
- API: Mais vitorioso e mais derrotado
- API: A partida com maior diferença de resultado
- API: Adicionar Saldo de gols, gols feitos e gols levados

Tarefas realizadas!

vicnetto committed:
Updating some files

vicnetto committed:
Novo commit

Tibianobr commented:
1. Revisar campo teamWinner no MatchEntity para referenciar o Team;
2. Usar modelo incremental nas migrações do banco;
3. Arrumar anotações nas entidades (@Column, @Basic);
4. Criar entidade de retorno no mostWinnerMostLoser para facilitar manipulação do front

vicnetto committed:
:art: Realizando alterações no formato da tabela MATCH

Agora, a tabela match teve um parâmentro alterado, que antes era uma
**String**, agora é uma TeamEntity. Isso foi feito para facilitar a
exibição no arquivo **JSON**, e facilitar o próprio código.

vicnetto committed:
:twisted_rightwards_arrows: Realizando merge branch 'estrutura-codigo' em vicnetto

Dentro do estrutura-código, foram realizadas alterações no formato da
tabela, como ao invés da variável teamWinner portar uma String, agora
tem posse de uma entidade TeamEntity. Isso foi realizado para facilitar
a visualização.

vicnetto committed:
:recycle: Refazendo o arquivo V2_TEAMS.sql e inserindo V5

O V2_TEAMS.sql foi retornado para o seu formato original, e para isso
foi adicionado o V5_TEAMS_ALTER.sql, promovendo uma alteração na tabela
feita no V2. Antes essa modificação tinha sido realizado de forma
errada, sendo inserida alterações nas versões ao invés de realizar o
versionamento correto.

vicnetto committed:
Inserindo nova classe de retorno mostWinnerMostLoser

Inserido nova classe utilizado para retorno, visando facilitar o
front-end. Dentro dessa classe agora existem dois atributos:
teamMostWinner e teamMostLoser. Além disso, foi inserido as devidas
colunas as váriaveis, melhorando a visualização do código.
